docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -1,35 +1,93 @@
+in command line. Input values are *divided* by the scale value(s). If also `--reverse_input_channels` option is used, the reverse_input_channels
+will be applied first, then mean and after that scale.
 
 There is no a universal recipe for determining the mean/scale values for a particular model. The steps below could help to determine them:
 * Read the model documentation. Usually the documentation describes mean/scale value if the pre-processing is required.
 * Open the example script/application executing the model and track how the input data is read and passed to the framework.
-* Open the model in a visualization tool and check for layers performing subtraction or multiplication (like `Sub`, `Mul`, `ScaleShift`, `Eltwise` etc) of the input data. If such layers exist, pre-processing is probably part of the model.
+* Open the model in a visualization tool and check for layers performing subtraction or multiplication (like `Sub`, `Mul`, `ScaleShift`, `Eltwise` etc)
+of the input data. If such layers exist, pre-processing is probably part of the model.
+
+For example, run the Model Optimizer for the PaddlePaddle* UNet model and apply mean-scale normalization to the input data.
+
+```sh
+mo --input_model unet.pdmodel --input data --mean_values data[123,117,104] --scale_values data[255,255,255]
+```
 
 ## When to Reverse Input Channels <a name="when_to_reverse_input_channels"></a>
-Input data for your application can be of RGB or BRG color input order. For example, OpenVINO Samples load input images in the BGR channels order. However, the model may be trained on images loaded with the opposite order (for example, most TensorFlow\* models are trained with images in RGB order). In this case, inference results using the OpenVINO samples may be incorrect. The solution is to provide `--reverse_input_channels` command line parameter. Taking this parameter, the Model Optimizer performs first convolution or other channel dependent operation weights modification so these operations output will be like the image is passed with RGB channels order.
+Input data for your application can be of RGB or BRG color input order. For example, OpenVINO Samples load input images in the BGR channels order.
+However, the model may be trained on images loaded with the opposite order (for example, most TensorFlow\* models are trained with images in RGB order).
+In this case, inference results using the OpenVINO samples may be incorrect. The solution is to provide `--reverse_input_channels` command line parameter.
+Taking this parameter, the Model Optimizer performs first convolution or other channel dependent operation weights modification so these operations output
+will be like the image is passed with RGB channels order.
+
+For example, launch the Model Optimizer for the TensorFlow* AlexNet model with reversed input channels order between RGB and BGR.
 
+```sh
+mo --input_model alexnet.pb --reverse_input_channels
+```
```

Comment:
can we add `see also` with preprocessing in runtime? It provides more capabilities

Reply:
Done